src/style.css

```diff
@@ -1,3 +1,62 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  width: 1200px;
+  background-color: fff;
+  justify-content: space-between;
+
```

Comment:
There is a missing closing bracket '}' for the '.header' class. Ensure that each CSS rule set is properly closed to avoid unexpected behavior.